cli/js/utime_test.ts

```diff
@@ -19,8 +19,8 @@ testPerm({ read: true, write: true }, function utimeSyncFileSuccess(): void {
   Deno.utimeSync(filename, atime, mtime);
 
   const fileInfo = Deno.statSync(filename);
-  assertFuzzyTimestampEquals(fileInfo.accessed, atime);
-  assertFuzzyTimestampEquals(fileInfo.modified, mtime);
+  assertFuzzyTimestampEquals(fileInfo.accessed!, atime);
```

Comment:
`assertFuzzyTimestampEquals` should accept a possibly undefined value on the left hand side...